File: src/main/java/com/vayber/pinyougou/bean/PayInfo.java
package com.vayber.pinyougou.bean;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import org.springframework.stereotype.Repository;

//支付信息
@Repository
@TableName("payinfo")
@Data
public class PayInfo {
    @TableId(type = IdType.AUTO)
    Integer id;
    String type;
    String name;
    String number;
    String billAddress;
    String expiryDate;
    Integer customerId;

}

File: pinyougou-vue/src/network/payInfo/payInfo.js
import {request} from '../request.js'

export function savePayInfo(payInfo,customerId) {
	return  request({
		url: '/payInfo/add',
		params:{
			type:payInfo.typeCard,
			name:payInfo.cardName,
			number:payInfo.cardId,
			billAddress:payInfo.billAddress,
			expiryDate:payInfo.expiryDate,
			customerId
		}
	})
}

export function getPayInfoById(customerId) {
	return  request({
		url: '/payInfo/query',
		params:{
			
			customerId
		}
	})
}